mezzanine/e-con-systems-camera/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Documentation e-con Systems Camera Mezzanine
+permalink: /documentation/mezzanine/e-con-systems-camera/
+---
+
+## e-con Systems Camera Mezzanine Card from e-con Systems
+
+# Table of Contents
+- [1) Introduction](#1-introduction)
+  - [1.1) Specifications & Features](#11-specifications-and-features)
+- [2) Guides and Documentation](#2-guides-and-documentation)
+
+# 1) Introduction
+
+<img src="https://github.com/96boards/website/blob/master/_product/mezzanine/e-con-systems-camera/images/e-con-systems-camera-front.jpg?raw=true" data-canonical-src="https://github.com/96boards/website/blob/master/_product/mezzanine/e-con-systems-camera/images/e-con-systems-camera-front.jpg?raw=true" width="300" height="250" />
```

Comment:
@kylekirkby can we reference this locally?

Reply:
The _product folder is something Jekyll reads as a "collection" so files that are not rendered as pages simply are not in the built site output. You can add images to the `assets/images` folder in the website repo and then include using a /assets/images/foldername/image.png path.